### PR TITLE
Create MERN_CRUD_Online_store.yml

### PR DESCRIPTION
**Pull request by aawgit**

### CODE REVIEW

File: _data/projects/MERN_CRUD_Online_store.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Online store app with MERN 
+desc: A boilerplate for a front and and back end API. An online store app is instantiated.
```
Comment (ritwik12) [inline _data/projects/MERN_CRUD_Online_store.yml]: @aawgit Please correct typo, also improve desc

File: _data/projects/MERN_CRUD_Online_store.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- ReactJS
+- MongoDB
+- Mongoose
+- Express
+- Cloudinary
+- Passport-facebook
+- Oauth
+upforgrabs:
+  name: good first issue
+  link: https://github.com/aawgit/onlineStore/blob/master/README.md
```
Comment (ritwik12) [inline _data/projects/MERN_CRUD_Online_store.yml]: @aawgit link should be https://github.com/aawgit/onlineStore/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/MERN_CRUD_Online_store.yml (hunk 2 above)
Comment (aawgit) [reply _data/projects/MERN_CRUD_Online_store.yml]: Thanks. Changes are done.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@aawgit Thanks!!

**Pull request merged**